base/zfp_compressor.hpp

```diff
@@ -5,7 +5,7 @@
 #include <vector>
 
 #include "base/not_null.hpp"
-#include "zfp/zfp.h"
+#include "zfp.h"
```

Comment:
We have automation that builds for Macintosh, Linux, and Windows, although it is not based on `install_deps.sh`; the `#include` is fine, if you need to change it it means the `install_deps` does not do what it should.

Reply:
Reverted in 53d3588 and fixed by using the right zfp repo.